### ISSUE TITLE
mapIsReady is called never if user decline location permissions

### ISSUE THREAD

**Issue by AlehSabadashGmail** [liodali/osm_flutter#348]
Hey everyone, there is an issue when you want to use map without location permission

Version: 0.40.3+2

**Comment by AlehSabadashGmail** [comment]
@liodali any thoughts here, maybe you know how to fix it quickly

Best regard

**Comment by liodali** [comment]
on which platform you faced this issue ?

**Comment by AlehSabadashGmail** [comment]
ios

**Comment by AlehSabadashGmail** [comment]
OSMFlutter(
          controller: controller,
          onMapIsReady: (isReady) {
            print('isReady'); 
            print(isReady); // always false
          },
        ),

**Comment by liodali** [comment]
yeah for ios now the permission is need it but i will find solution at leat to make the track not possible it permission not granted

**Comment by AlehSabadashGmail** [comment]
thank you very much, hope it will be soon, let me know if I can help you

**Comment by liodali** [comment]
i will start working on this issue this week

**Comment by liodali** [comment]
can you share with me your map_controller config ?

**Comment by AlehSabadashGmail** [comment]
sure, not a lot
```
    controller = MapController(
      initPosition: _initPosition,
    );
```

_initPosition - just return GeoPoint

**Comment by AlehSabadashGmail** [comment]
or I do not utunderstand you correctly, Does it what you want?

**Comment by liodali** [comment]
because permission it only required when initwithuserposition only or when you want to track the user

**Comment by liodali** [comment]
make iniwithUserPosition: false 
because default is true 
i will add named constructor to ony specify position or default ini with user position

**Comment by liodali** [comment]
check our new version `0.41.1` we added new named constructor `withPosition`you can used
